circle.yml

```diff
@@ -0,0 +1,13 @@
+machine:
+  java:
+    version: oraclejdk8
+
+deployment:
+  develop:
+    branch: develop
+    commands:
+      -  mvn deploy
+  release:
+    tag: /v[0-9]+(\.[0-9]+){0,2}(-rc)?/ # v1.2.3 or v1.2.3-rc
```

Comment:
I thought we decided we wanted to do branch based deploys, not tag/RC based deploys?